department-of-veterans-affairs/caseflow
# AMA hearing worksheet

Resolves #8475 

### Description
This PR updates the hearing worksheet to work for AMA appeals.

### Testing Plan
1. Create an AMA hearing, go to the hearing worksheet, confirm functionality works as expected
1. Create a legacy hearing, go to the hearing worksheet, confirm functionality works as expected
